doc/openshift-logging.adoc

```diff
@@ -0,0 +1,128 @@
+# Application Logging with Elasticsearch, Fluentd, and Kibana
```

Comment:
change title to Application Logging with Elasticsearch, Fluentd, and Kibana on Red Hat OpenShift Container Platform